packages/http-service/src/stream.js

```diff
@@ -0,0 +1,27 @@
+import Rx from "rxjs";
+
+import ConnectionManager from "connection-manager";
+import { XHRConnection, ConnectionEvent } from "connections";
+
+export default function stream(url, options = {}) {
+  return Rx.Observable.create(function(observer) {
+    const connection = new XHRConnection(url, options);
+
+    connection.addListener(ConnectionEvent.DATA, function(event) {
+      observer.next(event.target.response);
+    });
+    connection.addListener(ConnectionEvent.ERROR, function(event) {
+      observer.error(event);
+    });
+    connection.addListener(ConnectionEvent.COMPLETE, function() {
+      observer.complete();
```

Comment:
We probably should to push the data on complete as well before closing the steam.

Reply:
In this case we will be duplicating the last chunk and this is not something you want to have subscribing to a stream

Reply:
Is the data event also firing on complete?

Reply:
The thing is it's not `data` it's `progress`. Even when error occurs, first you get the `progress` event and then `error`

Reply:
Alright, thanks for checking.